lib/lookup.json

```diff
@@ -311,5 +311,13 @@
   },
   "node-report": {
     "prefix": "v"
+  },
+  "thread-sleep": {
+    "install": ["--build-from-source"]
+  },
+  "bufferutil": {
+    "npm": true
```

Comment:
Is there no way that we can fetch from github here?

looks like `"prefix":  "v"` should be okay